apps/server-asset-sg/src/app/jwt/jwt-middleware.ts

```diff
@@ -81,6 +82,16 @@ export class JwtMiddleware implements NestMiddleware {
         );
     }
 
+  private isAuthorizedByGroupE(jwt: Jwt): E.Either<Error, Jwt> {
+      const authorizedGroups = process.env.OAUTH_AUTHORIZED_GROUPS?.split(',');
+      if (!authorizedGroups) {
+        return E.left(new Error('An internal server error occurred. Please try again.'));
+      }
+      const userGroups = (jwt.payload as JwtPayload)['cognito:groups'] as string[];
```

Comment:
Similar to `authorizedGroups`: lowercase these groups for a case-insensitive comparison.
I assume that trimming is not be necessary here.

Reply:
added both the trim and toLowerCase just in case